setup-rules.md

```diff
@@ -58,3 +62,39 @@ Request must be submitted, reviewed, and merged before updating the
 superproject.
 
 [submodules]: https://www.git-scm.com/book/en/v2/Git-Tools-Submodules
+
+
+
+### How We Use Submodules
+
+Many of our products are dependent on other projects. We need two
+things when we build projects that make use of dependencies:
+
+1. Many of our builds require dependent projects to be in known
+specific locations so one project can trigger a build of a related
+project, and
+
```

Comment:
Just a minor suggestion: Do you think it would help to have examples here - like pointing to dependency on pal etc?
Or would it be an overkill?